### PR TITLE
Helm chart: add the posibility to add sidecars to Zulip

### PR DESCRIPTION
**Pull request by mikkeschiren**
Sidecars could have many uses, like adding exporters, scripts etc.

### CODE REVIEW

File: kubernetes/chart/zulip/values.yaml [hunk 1]
```diff
@@ -177,6 +177,17 @@ postSetup:
...
+  # -- You can add any sidecar to Zulip, like:
+  # sidecars:
+  #   - image: busybox
+  #     name: busy
+  #     command:
+  #       - "/bin/sh"
+  #       - "-c"
+  #       - "echo 'message from sidecar' > /inthis.txt"
+sidecars:
+  []
```
Comment (timabbott) [inline kubernetes/chart/zulip/values.yaml:189]: Is busybox a realistic sidebar? I'd like to have an example that would be natural for someone to want.

Comment (mikkeschiren) [reply kubernetes/chart/zulip/values.yaml:189]: That is just a custom placeholder just in many projects as just a placeholder, but I can add another example.

Comment (mikkeschiren) [reply kubernetes/chart/zulip/values.yaml:189]: Now I added a minio client as a  sidecar instead, I think that would make more sense?

File: kubernetes/chart/zulip/values.yaml [hunk 2]
```diff
@@ -177,6 +177,17 @@ postSetup:
     #   #!/bin/bash
     #   echo "This is a script that gets executed in the Zulip container after installation, once migrations are complete"
 
+  # -- You can add any sidecar to Zulip, like a minio client to use for
+  # uploading backups to a bucket, and mounting the volume for Zulip.
+  # sidecars:
```
Comment (timabbott) [inline kubernetes/chart/zulip/values.yaml:182]: This isn't new in this PR, but this comment style isn't very readable; should we think about switching to using the double-# pattern for the text part/
```
## text comment
# commented code
```

If you like the idea, just put up a follow-up PR for it.

**Comment by timabbott** [comment]
Merged, thanks @mikkeschiren!

File: kubernetes/chart/zulip/values.yaml (hunk 2 above)
Comment (mikkeschiren) [reply kubernetes/chart/zulip/values.yaml:182]: The commenting format is for [helm docs](https://github.com/norwoodj/helm-docs) - and as you have README.md.gotmpl that uses helm docs for the README - I think it should be good to keep it. Maybe I also should have added the change to the README generated by Helm docs as part of pull request?

**Pull request merged**